docs/develop/go/message-passing.mdx

```diff
@@ -316,12 +316,20 @@ if err != nil {
 
 ### Send an Update {#send-update-from-client}
 
+:::tip Stability
+
+In [Pre-release](/evaluate/development-production-features/release-stages#pre-release) (API is subject to change).
+
+Minimum recommended [Temporal Server version 1.25](https://github.com/temporalio/temporal/releases/tag/v1.25.0)
```

Comment:
Consider "Minimum Temporal Server version" unless there's a compelling reason otherwise to be less restrictive.